openshift/tests-extension/Makefile

```diff
@@ -80,6 +80,14 @@ pkg/bindata/catalog/catalog.go: $(shell find testdata/catalog -type f)
 	$(GO_BINDATA) -pkg catalog -o $@ -prefix "testdata/catalog" testdata/catalog/...
 	go fmt ./$(@D)/...
 
+bindata: test/extended/testdata/bindata.go
+test/extended/testdata/bindata.go: $(shell find test/extended/testdata -type f ! -name 'bindata.go')
+	mkdir -p $(@D)
+	$(GO_BINDATA) -nocompress -nometadata \
+		-pkg testdata -o $@ -prefix "testextended" \
+		-ignore "test/extended/testdata/bindata.go" test/extended/testdata/...
+	gofmt -s -w $@
+
```

Comment:
The other bindata is located in `pkg/bindata/`, not sure if this was intentional, or...?

Reply:
@tmshort Thanks
I just followed the way of the tests-private, and now I change to pkg/bindata.